desktop/build.gradle.kts

```diff
@@ -136,7 +138,7 @@ for (platform in PackrConfig.Platform.values()) {
                         " --classpath $jarFile" +
                         " --mainclass $mainClassName" +
                         " --vmargs Xmx1G " +
-                        (if (platform == PackrConfig.Platform.MacOS) jvmArgsForMac.joinToString(" ") {
+                        (if (platform == PackrConfig.Platform.MacOS) customJvmArgs.joinToString(" ") {
```

Comment:
Haha, no worries!

Change done. But TBH I'm not totally clear on how Packr works or what this statement does. Hopefully it works.

The type cast (in new commit) is vaguely dangerous, because theoretically I suppose the Gradle builder scope could implement a custom getter that chews up the `MutableList` I tried to assign and returns something weird— Keep a `val` if so— Probably not worth prematurely worrying about, though.